mods/ctf/ctf_modes/ctf_modebase/match.lua

```diff
@@ -53,6 +54,9 @@ function ctf_modebase.start_new_match(show_form)
 			voting = false
 
 			start_new_match()
+				
+			local new_mode = ctf_modebase.current_mode
+			RunCallbacks(ctf_modebase.registered_on_new_mode, new_mode, old_mode)
```

Comment:
```suggestion
			RunCallbacks(ctf_modebase.registered_on_new_mode, most.n, old_mode)
```

Reply:
Where is `old_mode` defined? If it isn't maybe put
```lua
local old_mode = ctf_modebase.current_mode
```
above
```lua
ctf_modebase.current_mode = most.n
```

Reply:
old mode is right below `old_map`